internal/util/util.go

```diff
@@ -191,3 +191,45 @@ func GetClients() (*kubernetes.Clientset, *snapshotterClientSet.Clientset, error
 
 	return client, snapshotterClient, nil
 }
+
+// IsVolumeSnapshotClassHasListerSecret returns whether a volumesnapshotclass has a snapshotlister secret
+func IsVolumeSnapshotClassHasListerSecret(vc *snapshotv1beta1api.VolumeSnapshotClass) bool {
+	nameExists := false
```

Comment:
no need to declare these ahead of time, I'd just declare them as part of trying to read from the Annotations map.